tests/possible_values.rs

```diff
@@ -196,6 +215,22 @@ fn possible_values_output() {
     ));
 }
 
+#[test]
+fn possible_values_hidden_output() {
+    assert!(utils::compare_output(
+        App::new("test").arg(
+            Arg::new("option")
+                .short('O')
+                .possible_values(&["slow", "fast"])
+                .possible_value(ArgValue::new("ludicrous speed"))
+                .possible_value(ArgValue::new("forbidden speed").hidden(true))
+        ),
+        "clap-test -O slo",
+        PV_ERROR,
+        true
+    ));
+}
```

Comment:
Need another test to test the help output of this `App`. Maybe using existing help string in `tests/help.rs`?